ivy/data_classes/container/experimental/elementwise.py

```diff
@@ -2975,3 +2975,88 @@ def digamma(
             map_sequences=map_sequences,
             out=out,
         )
+
+    @staticmethod
+    def static_sparsify_tensor(
+        tensor: Union[ivy.Container, ivy.Array, ivy.NativeArray],
```

Comment:
```suggestion
       x: Union[ivy.Container, ivy.Array, ivy.NativeArray],
```

Let's rename this to `x` because it's consistent with the  rest of the functions in ivy.